R/lancelot.R

```diff
@@ -1075,6 +1126,14 @@ lancelot_compare <- function(state, observed, pars) {
                                   pars$kappa_death_non_hosp, exp_noise)
   ll_deaths <- ll_nbinom(observed$deaths,
                          pars$phi_death_hosp * model_deaths_hosp +
+                           pars$phi_death_hosp * model_deaths_hosp_0_49 +
+                           pars$phi_death_hosp * model_deaths_hosp_50_54 +
+                           pars$phi_death_hosp * model_deaths_hosp_55_59 +
+                           pars$phi_death_hosp * model_deaths_hosp_60_64 +
+                           pars$phi_death_hosp * model_deaths_hosp_65_69 +
+                           pars$phi_death_hosp * model_deaths_hosp_70_74 +
+                           pars$phi_death_hosp * model_deaths_hosp_75_79 +
+                           pars$phi_death_hosp * model_deaths_hosp_80_plus +
```

Comment:
Here we should have just `pars$phi_death_hosp * model_deaths_hosp` or just 
```pars$phi_death_hosp * model_deaths_hosp_0_49 +
 pars$phi_death_hosp * model_deaths_hosp_50_54 +
 pars$phi_death_hosp * model_deaths_hosp_55_59 +
 pars$phi_death_hosp * model_deaths_hosp_60_64 +
 pars$phi_death_hosp * model_deaths_hosp_65_69 +
 pars$phi_death_hosp * model_deaths_hosp_70_74 +
 pars$phi_death_hosp * model_deaths_hosp_75_79 +
 pars$phi_death_hosp * model_deaths_hosp_80_plus

Reply:
Also none of the age disaggregated hospital death streams are contributing to the likelihood currently beyond this

Reply:
Ah, sorry about this. Done